src/main/java/org/blockjam/core/config/ConfigManager.java

```diff
@@ -35,6 +33,8 @@
 import java.net.URL;
 import java.nio.file.Files;
 
+import static com.google.common.base.Preconditions.checkNotNull;
```

Comment:
The IntelliJ Code Style by Google orders all static imports to bottom of the import section.